pico/tools/src/main/java/io/helidon/pico/tools/TypeTools.java

```diff
@@ -1110,6 +1111,7 @@ static String extractInjectionPointTypeInfo(TypeSignature sig,
                             || isListType(typeArgClassName)
                             || typeArgClassName.equals(Collections.class.getName())) {
                         // not handled
+                        assert (true);   // left here for debugging purposes
```

Comment:
Please remove all such cases. You can debug code without explicit lines. If statements with useless conditions are just an overhead.

Reply:
See my earlier comment. There is no overhead outside of "-ea" mode of the jvm.

Reply:
pushed